core/common/src/main/java/alluxio/network/TieredIdentityFactory.java

```diff
@@ -98,37 +108,50 @@ private static TieredIdentity fromScript() {
     if (!Configuration.containsKey(PropertyKey.LOCALITY_SCRIPT)) {
       return null;
     }
```

Comment:
Is this still possible now that there is a default value?

Reply:
not possible any more, removed this check